src/frame.js

```diff
@@ -0,0 +1,44 @@
+// this handles the rolls and adding them to each frame, it only knows about the one frame
+class Frame {
+  constructor() {
+    this.roll1 = 0;
+    this.roll2 = 0;
+    this.rollCount = 1;
```

Comment:
You could achieve this by having a rolls array that is either empty (at the start), then the first roll gets added, and then the second roll.